<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU SCHEDULER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;C:/Users/Omar Hatem/Desktop/back.jpgbackground-image&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="out_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>651</width>
      <height>131</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <row>
     <property name="text">
      <string>process </string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Time</string>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="ok_but">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7,121, 25);</string>
    </property>
    <property name="text">
     <string>THE GANNT CHART !</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>630</y>
      <width>325</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>325</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7,121, 25);</string>
    </property>
    <property name="text">
     <string>AVERAGE WAITING TIME:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="waiting_time">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>680</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color = rgb(170,0,0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>651</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(3) Enter the required data below :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="inputTable">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="quantum">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mylabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter the Quantum time :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>551</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="fcfs_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FCFS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sjf_preemitive_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SJF(preemtive)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sjf_non_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SJF(non-preemitive)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="priority_pre_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Priority(preemitive)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="priority_non_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Priority(non-preemitive)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="roundrobin_but">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Round Roubin</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>611</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(2) Please select the scheduler Algorithm :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>610</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(1) Enter the number of processes: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="process_num">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
